src/main/kotlin/org/kamiblue/client/module/modules/combat/AutoMend.kt

```diff
@@ -124,10 +163,18 @@ internal object AutoMend : Module(
         return slot
     }
 
+    private fun SafeClientEvent.hasMending(slot: Int): Boolean {
+        val stack = player.inventory.armorInventory[slot]
+        return EnchantmentHelper.getEnchantmentLevel(Enchantments.MENDING, stack) > 0
+    }
+
     private fun SafeClientEvent.shouldMend(i: Int): Boolean { // (100 * damage / max damage) >= (100 - 70)
         val stack = player.inventory.armorInventory[i]
-        val hasMending = EnchantmentHelper.getEnchantmentLevel(Enchantments.MENDING, stack) > 0
-        return hasMending && stack.isItemDamaged && 100 * stack.itemDamage / stack.maxDamage > reverseNumber(threshold, 1, 100)
+        return hasMending(i) && stack.isItemDamaged && 100 * stack.itemDamage / stack.maxDamage > reverseNumber(threshold, 1, 100)
+    }
+
+    private fun SafeClientEvent.shouldMend(): Boolean {
+        return shouldMend(0) || shouldMend(1) || shouldMend(2) || shouldMend(3)
     }
```

Comment:
Perhaps combine these or reorganize it? Not that important but `shouldMend()` should be at the top of these, and `hasMending` + `shouldMend(i: Int)` could be combined without too much of an issue

Reply:
like it was originally? lol